core/blockchain.go

```diff
@@ -416,9 +416,25 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, chainConfig *par
 			triedb.SaveCachePeriodically(bc.cacheConfig.TrieCleanJournal, bc.cacheConfig.TrieCleanRejournal, bc.quit)
 		}()
 	}
+
+	// load the latest dirty accounts stored from last stop to cache
+	bc.loadLatestDirtyAccounts()
+
 	return bc, nil
 }
 
+func (bc *BlockChain) loadLatestDirtyAccounts() {
+	for i := uint64(dirtyAccountsCacheLimit); i > 0; i-- {
+		hash := rawdb.ReadCanonicalHash(bc.db, bc.CurrentBlock().NumberU64()-i)
```

Comment:
We want to store the dirty accounts of side chain, with this logic, dirty accounts of side chain are not loaded. I think we store them as an array of struct { blockHash, dirtyAccounts }